src/components/about/WhatMakesUsUnique.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { Grid, Typography } from '@material-ui/core';
+import { makeStyles } from '@material-ui/styles';
+
+import { spacing, fontSizing, useStylesBase } from '../styledComponents';
+
+import image1 from '../../assets/aboutPage/makesUsUnique.block1.jpg';
+import image2 from '../../assets/aboutPage/makesUsUnique.block2.jpg';
+import image3 from '../../assets/aboutPage/makesUsUnique.block3.jpg';
+
+const cardList = [
+  { title: 'Coding', subTitle: 'We engage engineers', image: image1 },
+  { title: 'Open-source', subTitle: 'We foster collaboration', image: image2 },
+  { title: 'Micro-volunteering', subTitle: 'we accelerate change', image: image3 },
+];
+
+const useStyles = makeStyles({
+  cardsContainer: {
+    display: 'flex',
+    position: 'relative',
+    width: 'calc(100vw - 8px)',
+    left: 'calc(-1 * (100vw - 100%) / 2)', // to centered the div
```

Comment:
Does this work responsively? Will it get weird if the page gets resized?

Reply:
Yes that is what I was talking about with Eric a few days before.
 I changed it :smile: